hoomd/logging.py

```diff
@@ -22,6 +31,7 @@ class LoggerCategories(Flag):
     quantities, and custom actions (`hoomd.custom.Action`).
 
     Flags:
+
         scalar: `float` or `int` objects (i.e. numbers)
```

Comment:
This section renders weird, there are a bunch of constants set to powers of 2 rendering in the docs page without any explanation around them, and they wrap the `any` method of this class.

Reply:
Sphinx is extracting the values of the enum names from the class. These are class members, so it is consistent to document them this way. @b-butler do you have a strong preference to override Sphinx here and provide a manually defined `Attributes:` section?

Reply:
Manually overwriting the `Attributes` section makes sense.

Reply:
Could you write that update @b-butler ?

Reply:
I can. I meant to do this Friday but slipped past me.